helical/models/geneformer/model.py

```diff
@@ -186,3 +187,59 @@ def ensure_data_validity(self, adata: AnnData, gene_names: str) -> None:
             If the data is missing column names.
         """
         self.ensure_rna_data_validity(adata, gene_names)
+
+    def fine_tune_classifier(
+            self,
+            train_dataset: Dataset, 
+            optimizer: optim = optim.AdamW,
+            optimizer_params: dict = {'lr': 0.0001}, 
+            loss_function: loss = loss.CrossEntropyLoss(), 
+            label: str = "cell_types", 
+            epochs: int = 1,
+            freeze_layers: int = 0,
+            validation_dataset: Optional[Dataset] = None,
+            lr_scheduler_params: Optional[dict] = None) -> BertForSequenceClassification:
+        """Fine-tunes the Geneformer model for classification tasks. 
```

Comment:
Ok, this is me nitpicking, but I think it'd be nice if you could add the Return value section as part of the doc strings as well as properly specifying the default value for optimizer, loss function etc. Cause I think the docs are then automatically generated based on these (so need to be formatted properly). Also, it'd be cool to explain why we're returning a BertForSequenceClassification model. End of my nitpicking.

Reply:
Updated!